pyinfra/modules/yum.py

```diff
@@ -107,10 +109,11 @@ def rpm(state, host, source, present=True):
 
     .. code:: python
 
+        linux_major = host.fact.linux_distribution['release_meta'].get('VERSION_ID')
         yum.rpm(
            {'Install EPEL rpm to enable EPEL repo'},
            'https://dl.fedoraproject.org/pub/epel/epel-release-latest-'
-           '{{  host.fact.linux_distribution.major }}.noarch.rpm',
+           '{}.noarch.rpm'.format(linux_major),
```

Comment:
I'm not sure why this change is needed? `host.fact.linux_distribution.major` should return the same value?

Reply:
Sorry. I misunderstood the jinja2 comment above. Reverted.